config/main.py

```diff
@@ -112,6 +113,40 @@ def load(filename):
     command = "{} -j {} --write-to-db".format(SONIC_CFGGEN_PATH, filename)
     run_command(command, display_cmd=True)
 
+@cli.command()
+@click.option('-y', '--yes', is_flag=True, callback=_abort_if_false,
+                expose_value=False, prompt='Reload mgmt config?')
+@click.argument('filename', default='/etc/sonic/device_desc.xml', type=click.Path(exists=True))
+def load_mgmt_config(filename):
+    """Reconfigure mgmt interface based on device description file."""
+    command = "{} -M {} --write-to-db".format(SONIC_CFGGEN_PATH, filename)
+    run_command(command, display_cmd=True)
+    #FIXME: After config DB daemon for mgmt interface is implemented, we'll no longer need to manually config mgmt interface here
+    mgmt_conf = parse_device_desc_xml(filename)['minigraph_mgmt_interface']
+    command = "ifconfig eth0 {} netmask {}".format(str(mgmt_conf['addr']), str(mgmt_conf['mask']))
+    run_command(command, display_cmd=True)
+    command = "[ -f /var/run/dhclient.eth0.pid ] && kill `cat /var/run/dhclient.eth0.pid` && rm -f /var/run/dhclient.eth0.pid"
+    run_command(command, display_cmd=True)
```

Comment:
No. Because interfaces-config service will read data from minigraph but not device description file.